ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -540,6 +555,49 @@ then
 		fi
 	fi
 
+	#
+	# MAVLink onboard / TELEM2
+	#
+	if ver hwcmp MINDPX_V2
+	then
+	else
+		# XXX We need a better way for runtime eval of shell variables,
+		# but this works for now
+		if param compare SYS_COMPANION 921600
```

Comment:
No, we don't need to have this separately.  I've removed the codes.  Thanks Lorenz.